src/components/RunHeader/RunHeader.js

```diff
@@ -44,16 +44,23 @@ class RunHeader extends Component {
           }
           return (
             runName && (
-              <h1>
-                <div className="block-icon">
-                  {getStatusIcon({ reason, status })}
-                </div>
-                <div className="run-name" title={runName}>
-                  {runName}
-                </div>
-                <span className="status-label">{reason}</span>
-                <span className="time">{lastTransitionTime}</span>
-              </h1>
+              <div>
```

Comment:
Is the wrapping `div` here used to ensure the block extends the full width? If so it would be better to achieve this via explicit styles in the SCSS rather than adding the extra wrapper, this will make it easier to handle the responsive layout at smaller screen sizes.

Reply:
I'll run with this as well

Reply:
I think I added this so that it was all in one div to return as a jsx element and it wasn't to do with the block - but I don't even need it

Just built and tested, looks great without it anyway so I've removed it